inputs/worker_inner:6.json

```diff
@@ -3,6 +3,10 @@
     {
       "name": "reactor_config"
     },
+    {
+      "name": "check_and_set_platforms",
```

Comment:
IIRC, this was added because `pull_base_image` was unnecessarily validating base image in worker build. Since we turned off validation there now, can this plugin be removed?

Reply:
Fixed.